src/main/scala/org/broadinstitute/dsde/firecloud/model/Profile.scala

```diff
@@ -29,25 +29,20 @@ case class BasicProfile (
     title: String,
     contactEmail: Option[String],
     institute: String,
-    institutionalProgram: String,
```

Comment:
It may be nice to also remove these fields from FireCloud UI. Removing them here doesn't break anything in FireCloud UI, but they won't be honored anymore when filled out.